jobs/rocoto/wavepostpnt.sh

```diff
@@ -19,10 +17,8 @@ export jobid="${job}.$$"
 echo
 echo "=============== START TO RUN WAVE_POST_PNT ==============="
 # Execute the JJOB
-${HOMEgfs}/jobs/JGLOBAL_WAVE_POST_PNT
+"${HOMEgfs}/jobs/JGLOBAL_WAVE_POST_PNT"
 status=$?
-if [[ ${status} -ne 0 ]]; then
-    exit "${status}"
-fi
+if [[ "${status}" -ne 0 ]]; then exit "${status}"; fi
```

Comment:
This should stay on multiple lines. Check other script updates to make sure these checks are across multiple lines. See work done in #3507. Should these become `err` as well here?